### PR TITLE
Add Tasker

### PR DESCRIPTION
**Pull request by guifelix**
[https://github.com/guifelix/awesome-tasker](https://github.com/guifelix/awesome-tasker)

This list is about resources and apps related to Tasker. Tasker is an app that helps automate your android phone and this list curates resources to help people learn how to use Tasker

- I reviewed the following PRs: [1865](https://github.com/sindresorhus/awesome/pull/1865) & [1866](https://github.com/sindresorhus/awesome/pull/1866).
- My list has 55 awesome-lint errors. Awesome lint identifies google play links as double-link so I reported this issue: [122](https://github.com/sindresorhus/awesome-lint/issues/122).

### CODE REVIEW

**Comment by guifelix** [comment]
unicorn

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by tomhuang12** [comment]
Hi @guifelix , great list by the way! I would recommend using [awesome-lint](https://github.com/sindresorhus/awesome-lint) to check the mistakes you might have missed from the guidelines. Good luck!

**Comment by danforbes** [comment]
I think it would be nice to add resources for people who would like to develop their own Tasker tasks. Links to GitHub repos would also be nice.

**Comment by guifelix** [comment]
@tomhuang12 Thanks for the compliment! but as for the awesome-lint tip, if you check the first comment you'll see that I've already used it at there's also a PR for false positives.

@danforbes Yes it would be very nice, but unfortunately I've never seen a relevant Github repository about it yet

@sindresorhus To make it easier, I'll list below the guidelines with my answers/checks:

- **You have to review at least 2 other open pull requests**
as mentioned on the first comment, I've reviewed  1865 & 1866.
- **You have read and understood the instructions for creating a list**
yes
- **This pull request has a title in the format Add Name of List**
yes, the title is ```Add Tasker```
- **Your entry here should include a short description about the project/theme of the list. It should not describe the list itself. The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.**
the description is ```Android Automation app.```
- **Your entry should be added at the bottom of the appropriate category.**
yes, it is at the bottom
- **Has been around for at least 30 days.**
the list has 2 years now
- **Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it.**
the guidelines are ready
- **Run awesome-lint on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please report it.**
as mentioned on the first comment, I've ran awesome-lint and reported the false positive
- **The default branch should be named main, not master.**
the default branch is main
- **Includes a succinct description of the project/theme at the top of the readme. **
check, there is a succinct description on the project top of readme
- **It's the result of hard work and the best I could possibly produce.**
check
- **The repo name of your list should be in lowercase slug format:**
check, the repo is ```awesome-tasker```
- **The heading title of your list should be in title case format**
check, the heading title is ```Awesome Tasker```
- **Non-generated Markdown file in a GitHub repo**
check
- **The repo should have awesome-list & awesome as GitHub topics**
check
- **Not a duplicate**
check
- **Only has awesome items. Awesome lists are curations of the best, not everything**
check
- **Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs**
check
- **Includes a project logo/illustration whenever possible; Either centered, fullwidth, or placed at the top-right of the readme; The image should link to the project website or any relevant website; The image should be high-DPI. Set it to maximum half the width of the original image**
check; check; check
- **Entries have a description, unless the title is descriptive enough by itself**
check
- **Includes the Awesome badge; Should be placed on the right side of the readme heading; Should link back to this list**
check; check; check
- **Has a Table of Contents section; Should be named Contents, not Table of Contents; Should be the first section in the list; Should only have one level of nested lists, preferably none**
check; check; check; check
- **Has an appropriate license; Place a file named license or LICENSE; Do not add the license name or text to the readme; To verify that you've read all the guidelines, please comment on your pull request with just the word unicorn**
check (it has CC0); check; check; check (2nd comment)
- **Has contribution guideline; The file should be named contributing.md**
check; check
- **Has consistent formatting and proper spelling/grammar; The link and description are separated by a dash; The description starts with an uppercase character and ends with a period; Consistent and correct naming**
check; check; check
- **Doesn't use hard-wrapping**
check 
- **Doesn't include a Travis badge**
check
- **Doesn't include an Inspired by awesome-foo or Inspired by the Awesome project kinda link at the top of the readme**
check

File: readme.md [hunk 1]
```diff
@@ -707,6 +707,7 @@
 - [Biomedical Information Extraction](https://github.com/caufieldjh/awesome-bioie#readme) - How to extract information from unstructured biomedical data and text.
 - [Web Archiving](https://github.com/iipc/awesome-web-archiving#readme) - An effort to preserve the Web for future generations.
 - [WP-CLI](https://github.com/schlessera/awesome-wp-cli#readme) - Command-line interface for WordPress.
+- [Tasker](https://github.com/guifelix/awesome-tasker#readme) - Android Automation app.
```
Comment (sindresorhus) [inline readme.md:710]:
```suggestion
- [Tasker](https://github.com/guifelix/awesome-tasker#readme) - Android automation app.
```

**Comment by sindresorhus** [comment]
I think it would be better for the header image to include "Tasker". Then you could also drop the text header.

---

Linkify `Tasker` in `Tasker is an application for Android` to its website.

---

There's a lot of incorrect capitalizations in the descriptions. For example, "Official Wiki." => "Official wiki."

---

I would also remove the badges. They add nothing as that information is already clear in the GitHub interface.

---

I would like to see more than just plugins. Include some blog posts, tutorials, links to community resources (Stack Overflow tag, IRC, chat, forums, communities, Reddit, etc. Example: https://github.com/sindresorhus/awesome-electron#community), videos of talks, book, learning material, etc.

**Comment by sindresorhus** [comment]
Bump

**Comment by klark1kent** [comment]
It's not a bad list but I can see where the feedback makes sense.  For example the official plugins section has all the AutoApps listed but in reality Tasker includes plugins by default not listed. 

I'm reviewing this on my phone so I apologize if they are included  but I'm somewhat surprised you don't have other things on this list ..

1. These are just the other plugins I have installed. 
- Snackbar Tasker Plugin
- CF.lumen
- Kinetic Scroll
- Bouncer
- Secure Settings
- Sync Pro
- Termux:Tasker
- openHAB
- TinyCam PRO
- SharpTools
- Memento Database
- BBS

2. Taskernet Collector - arguably the biggest contribution to Tasker this year outside of what Joao has done. 

pghant.github.io/taskernet-collector


It's a good start though op. Get it right so others can start contributing. Also... Joao has a couple Repos on GitHub you may want to include. Not super current but still

**Comment by guifelix** [comment]
- @sindresorhus So sorry I took time to answer this, I had personal issues to deal with;
- I've added `Tasker` on the image as suggested;
- Linkified `Tasker` as suggested;
- Fixed capitalizations where it applied;
- Removed badges (with an exception of the `awesome` that is required);
- Added more content as suggested:
	- Reddit channels
	- StackOverflow Tag
	- Telegram Groups
	- Facebook Pages & Groups
	- Blog
	- Books
	- Videos
	- Assets

**Comment by mandievs** [comment]
https://github.com/sindresorhus/awesome/pull/1867#issuecomment-821970340

**Comment by mandievs** [comment]
https://github.com/sindresorhus/awesome/pull/1867#issuecomment-821970340

**Comment by mandievs** [comment]
https://github.com/sindresorhus/awesome/pull/1867#issuecomment-821970340

**Pull request closed without merging**